src/components/CreateButton.tsx

```diff
@@ -27,26 +33,30 @@ function showSubCreateButtons(): JSX.Element {
 }
 
 export const CreateButton = (): JSX.Element => {
-  const [isShowSubButtons, setIsShowSubButtons] = useState<boolean>(false);
+  const [showSubButtons, setShowSubButtons] = useState(false);
+  const [showForm, setForm] = useState(false);
```

Comment:
change to `setShowForm`?
```
const [showForm, setShowForm] = useState(false);
```

Reply:
updated on https://github.com/RR-Ardhika/murojaah/pull/6/commits/b37e80f22115bbf144f05ac22ad27dfd2107e82a thank you